src/allocator/memserver_allocator.cpp

```diff
@@ -107,6 +129,30 @@ void Memserver_Allocator::reset_profile() {
     MEMSERVER_PROFILE_START_TIME(NVMM)
 }
 void Memserver_Allocator::dump_profile() { NVMM_PROFILE_DUMP(); }
+
+void Memserver_Allocator::garbage_collector_th(uint64_t thread_index)
```

Comment:
change name from garbage_collector to delayed free.

Reply:
Will take care of this as well as the main method in NVMM name also to be changed.